README.md

```diff
@@ -5,5 +5,5 @@ Basic Usage
 
 These examples demonstrate initialization and basic usage. They have the bare minimum code needed.
 
-- [`glfw31-gl41core-cube`](./glfw31-gl41core-cube/) - Renders a textured spinning cube using GLFW 3.1 and OpenGL 4.1 core forward-compatible profile.
-- [`glfw31-gl21-cube`](./glfw31-gl21-cube/) - Renders a textured spinning cube using GLFW 3.1 and OpenGL 2.1.
+-	[`gl41core-cube`](gl41core-cube) - Renders a textured spinning cube using GLFW and OpenGL 4.1 core forward-compatible profile.
+-	[`gl21-cube`](gl21-cube) - Renders a textured spinning cube using GLFW and OpenGL 2.1.
```

Comment:
Maybe we should include the major (but not minor) version here? i.e. "using GLFW 3 and OpenGL ..."